contrib/julia.xml

```diff
@@ -198,7 +196,6 @@
       <item> UInt128 </item>
       <item> Union </item>
       <item> Unsigned </item>
-      <item> UTF8String </item>
```

Comment:
Not completely related, but I just noticed `AbstractString` is missing from the list (hence the presence of the old `String` abstract type).